docs/getting-started.md

```diff
@@ -53,7 +53,7 @@ project repository, or, if there is no `.ort.yml` in the repository, its content
 
 Note that the file to which we refer here as the project's `ort.yml` file can have a name specific to the artifact or
 release or flavor, provided that it ends with ".ort.yml", for example it can be named "my-artifact.ort.yml". This means that multiple
-`*.ort.yml` files can exist in a repository to support different, each specific to a release or flavor of the software. 
+`*.ort.yml` files can exist in a repository, each specific to a release or flavor of the software. 
```

Comment:
Please drop the merge commit, do a rebase instead.

Reply:
I don't know how to do that, Frank

Reply:
Mind joining ORT slack chat and ping me? I could help you with it there...